todo-app/app/views/tasks/index.html.haml

```diff
@@ -0,0 +1,20 @@
+%h1 Todo List
+
+%table{ class: 'table' }
+  %tr
+    %th Title
+    %th Details
+    %th Complete?
+
+  - @tasks.each do |t|
+    %tr
+      %td= t.title
+      %td
+        %a{ href: "/tasks/#{t.id}" } Details
```

Comment:
This doesn't feel very Rails-ian.  `link_to` and `task_path` or something like that?  It's been a while.